content/soarian/r4/individuals/patient.md

```diff
@@ -0,0 +1,162 @@
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_SEARCH_BY_ID_ENTRY) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Errors
+
+* The common [errors] and [OperationOutcomes](https://www.hl7.org/fhir/r4/operationoutcome.html) may be returned. 
+* In addition, a `422 Unprocessable Entity` HTTP response status code and OperationOutcome is returned when >499 patients qualify for the search criteria.
+
+## Retrieve by ID
+
+Return an individual Patient resource by the resource ID:
+
+    GET /Patient/:id
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true) %>
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient/A879904FD2FE4B2D90C89FDA84E1285F
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_READ_BY_ID_ENTRY) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Linking Duplicate Patient Records Example
+
+The **link** element is used to assert that patient resources refer to the same patient. This element is used to support the scenario where a duplicate record is retired; this process is known as a **patient merge** in Soarian Clincials<sup>®</sup>. When the requested patient record has been merged into another record, an inactive Patient entry is returned that has a link to the current Patient entry. Entries for merged, or source, patients are only returned when retrieving the entries directly by ID. Entries for the target patient are returned with all qualifying searches.
+
+The ability to perform patient merge operations is not available through the Cerner Ignite APIs for Soarian Clinicals<sup>®</sup> platform..
+
+#### Request
+
+    GET https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient?_id=0A1119CA2414486AAD618F5A2D25556B
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_MERGE) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Errors
+
+The common [errors] and [OperationOutcomes](https://www.hl7.org/fhir/r4/operationoutcome.html) may be returned.
+
+[search]: https://www.hl7.org/fhir/http.html#search
+[`date`]: https://hl7.org/fhir/R4/search.html#date
+[`string`]: https://hl7.org/fhir/R4/search.html#string
+[`token`]: http://hl7.org/fhir/R4/search.html#token
+[`_revinclude`]: https://www.hl7.org/fhir/search.html#revinclude
+[errors]: ../../#client-errors
+[OperationOutcomes]: https://hl7.org/fhir/R4/operationoutcome.html
+[US Core Race]: https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-race.html
```

Comment:
is there a reason we're pointing to the build version, and not the published version?
https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-race.html

Reply:
Modified to published version